Bottom: dd0a3babc2c0d4eff81d89d31a763b75cd824ccc
Top:    3ff87638555ade345486c3fbfb01009cd769bbd7
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2012-11-15 01:46:48 +0000

Add warning about bogus code in strflags.c


---

diff --git a/src/strflags.c b/src/strflags.c
index 6cb6548..e6ce175 100644
--- a/src/strflags.c
+++ b/src/strflags.c
@@ -575,6 +575,7 @@ dump_flag (FlagType * f)
 {
   int l;
   printf ("F:%08x T:[", f->f);
+#warning THIS NEXT LINE IS BOGUS, THE ARRAY IS HALF THIS LENGTH
   for (l = 0; l < (MAX_LAYER + 7) / 8; l++)
     printf (" %02x", f->t[l]);
   printf ("]");
